Added sample codes for API:Embeddedin

edyasikpo opened the pull request:
API:Embeddedin gets a list of all page(s) that embed (transclude) the given title. 
My sample code displays all pages that embed the English Wikipedia's page on [w:Computer](https://en.wikipedia.org/wiki/Computer).
1) I have added the Get requests params to  modules.json.
2) The file is auto generated.
3) I ran my codes through pylint and it passed all the Pylint tests.
4) Added code and page description to  README.md 
This is the [link](https://www.mediawiki.org/wiki/User:Didicodes/Sandbox/API:Embeddedin) to the sandbox page I created for this API
Kindly review @srish

srish commented:
@edyasikpo thanks for your contribution! the page you worked on has been translated in many different languages. And, that is why it needs to be handled with care :) 

I see that you've added translation markers like <!-- T1--> and<!--T2--> to new sections. These tags are actually added by the translation software, and so we should not add them manually. I would encourage you to read this tutorial first: https://www.mediawiki.org/wiki/Help:Extension:Translate/Page_translation_example especially the section https://www.mediawiki.org/wiki/Help:Extension:Translate/Page_translation_example#Step_4:_Making_changes and make any necessary changes in your Sandbox. I will take another look then! Thanks!

edyasikpo commented:
@srish  Thank you for your feedback. I will read through the tutorial and make the necessary changes soon.

edyasikpo commented:
Hello, @srish I have made the changes you requested for on API:Embeddedin
I removed all the translation makers and added only the `<translate> </translate>` tags. I posted it on Zulip some days ago but it seems posting it here on GitHub will make it easier for you to find and also review the changes. This is the link to my sandbox: https://www.mediawiki.org/wiki/User:Didicodes/Sandbox/API:Embeddedin
Please review!

srish commented:
@edyasikpo Thanks for making the changes :) It looks like you got it now how to deal with pages with translations ;-) It is ready to go in the main namespace now!
